client/my-sites/activity/activity-log-v2/index.tsx

```diff
@@ -45,18 +89,12 @@ const ActivityLogV2: FunctionComponent = () => {
 			} ) }
 			wideLayout={ ! isJetpackCloud() }
 		>
+			{ siteId && <QuerySitePlans siteId={ siteId } /> }
 			<DocumentHead title={ translate( 'Activity log' ) } />
 			<SidebarNavigation />
 			<PageViewTracker path="/activity-log/:site" title="Activity log" />
 			{ isJetpackCloud() ? (
```

Comment:
Out of curiosity, don't we want to provide the upgrade path in WordPress.com as well?

Reply:
Yes good catch, I switched it to be the same as the WordPress.com upgrade path.

Reply:
Sorry Robert, I didn't express myself well and my question wasn't clear. There is a flag called `activity-log/v2` that lets you display the new version of the Activity Log (the same we have in the cloud.jetpack.com) in WordPress.com. If I activate that flag by adding `?flags=activity-log/v2` to end of the URL, I don't see the upgrade path. I only see it in cloud.jetpack.com.

So, my question was whether that is expected.
![image](https://user-images.githubusercontent.com/3418513/86833253-ab88e600-c06f-11ea-914a-78a782371a74.png)